### ISSUE TITLE
Wrong params for makeCallingParams

### ISSUE THREAD

**Issue by nikis** [feathers-plus/batch-loader#7]
When using loaderFactory method is calling custom makeCallingParams which create new context. By the default this function shoud include ['provider', 'authenticated', 'user'] from given context, but this not happening because this properites is trying to get from context not from context.params. 

```js
if (include) {
    (Array.isArray(include) ? include : [include]).forEach(name => {
      if (name in context) included[name] = context[name];
    });
  }
```

to work like original makeCallingParams need to change to this:

```js
if (name in context.params) included[name] = context.params[name];
```

or just to use [makeCallingParams](https://feathers-plus.github.io/v1/feathers-hooks-common/#makecallingparams)

**Comment by eddyystop** [comment]
Changed it to
```js
if (context.params && name in context.params) included[name] = context.params[name];
```
There remains an incompatibility in that BatchLoader version includes `paginate: false` as a default.

People can seemingly delay upgrading Feathers versions as long as possible. So I'd prefer leaving batch-loader woking with v4 rather than force an update to v4 feathers-hooks-common.

Published as v0.3.6

**Comment by eddyystop** [comment]
Closing. Feel free to reopn if necessary.
